modules/distr-tracing-tempo-config-sampling.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
```

Comment:
There is  no sampling options currently on tempo. I think this will be controlled by the otel collector.